### PR TITLE
i18n(zh-cn): Update docs for 4.13

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

Update docs for 4.13

Merge this with https://github.com/withastro/docs/pull/8969

#### Related issues & labels (optional)

- https://github.com/withastro/docs/pull/8967
- https://github.com/withastro/docs/pull/8995
- https://github.com/withastro/docs/pull/8943
- https://github.com/withastro/docs/pull/8999
- https://github.com/withastro/docs/pull/9006

### CODE REVIEW

File: src/content/docs/zh-cn/guides/middleware.mdx [hunk 1]
```diff
@@ -195,6 +196,80 @@ export const onRequest = sequence(validation, auth, greeting);
...
+
+使用 `context.rewrite()` 在中间件中显示不同页面的内容，而不会将访问者 [重定向](/zh-cn/guides/routing/#动态重定向) 到新页面。这将触发一个新的渲染阶段，导致任何中间件被重新执行。
+
+```js title="src/middleware.js"
+import { isLoggedIn } from "~/auth.js"
+export function onRequest (context, next) {
+  if (!isLoggedIn(context)) {
+    // 如果用户未登录，则更新请求来渲染 `/login` 路由，
+    // 并添加标头以指示在成功登录应该把用户重定向到何处。
+    // 重新执行中间件
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/guides/middleware.mdx:214]:
```suggestion
    // 重新执行中间件。
```
对比源文档掉了一个小标点~

File: src/content/docs/zh-cn/reference/error-reference.mdx [hunk 2]
```diff
@@ -108,5 +108,7 @@ githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/
 - [**DataCollectionEntryParseError**](/zh-cn/reference/errors/data-collection-entry-parse-error/)<br/>Data collection entry failed to parse.
 - [**DuplicateContentEntrySlugError**](/zh-cn/reference/errors/duplicate-content-entry-slug-error/)<br/>Duplicate content entry slug.
 - [**ActionsWithoutServerOutputError**](/zh-cn/reference/errors/actions-without-server-output-error/)<br/>Actions must be used with server output.
+- [**ActionsUsedWithForGetError**](/zh-cn/reference/errors/actions-used-with-for-get-error/)<br/>An invalid Action query string was passed by a form.
+- [**ActionQueryStringInvalidError**](/zh-cn/reference/errors/action-query-string-invalid-error/)<br/>An invalid Action query string was passed by a form.
 - [**UnsupportedConfigTransformError**](/zh-cn/reference/errors/unsupported-config-transform-error/)<br/>Unsupported transform in content config.
 - [**RouteNotFound**](/zh-cn/reference/errors/route-not-found/)<br/>Route not found.
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/error-reference.mdx:114]:
```suggestion

```
对比源文档发现这一行已经被移除了，但是我没有追溯到删除的改动历史。

File: src/content/docs/zh-cn/reference/errors/rewrite-encountered-an-error.mdx [hunk 3]
```diff
@@ -4,6 +4,10 @@ i18nReady: true
 githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
 ---
 
+:::caution[废弃]
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/errors/rewrite-encountered-an-error.mdx:7]:
```suggestion
:::caution[已废弃]
```
其他 caution 组件的警示词也是 `已废弃`

File: src/content/docs/zh-cn/reference/errors/actions-used-with-for-get-error.mdx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+title: An invalid Action query string was passed by a form.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **ActionsUsedWithForGetError**: 使用 GET 请求从表单调用 Action ACTION_NAME，但只支持 POST 请求。这通常是因为表单上缺少 `method="POST"`。
+
+## 哪里出了问题？
+
+Action 被使用 GET 请求从表单调用，但只支持 POST 请求。这通常是因为表单上缺少 `method="POST"`。
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/errors/actions-used-with-for-get-error.mdx:11]:
```suggestion
使用 GET 请求从表单中调用 Action，但只支持 POST 请求。这通常是因为表单上缺少 `method="POST"`。
```
瑞丰，这样修改会更好一些吗？

**Review by Nin3lee — CHANGES_REQUESTED** [review]
一些小改动和句序调整。

**Review by Nin3lee — APPROVED** [review]
LGTM!

**Comment by yanthomasdev** [comment]
Thanks @liruifengv, I believe we're missing both these commits that have been added after your PR: https://github.com/withastro/docs/commit/82b728323fdacb08eb35628bd8e41963773a8880 https://github.com/withastro/docs/commit/8cbc423fefad7c41eacf227bc6b51f142c89ec0f

**Comment by liruifengv** [comment]
Thank you Yan! @yanthomasdev I have update all.

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good job!

**Pull request merged**